packages/ocaml-base-compiler/ocaml-base-compiler.4.07.0/opam

```diff
@@ -23,8 +23,8 @@ build: [
     "-aspp"
     "cc -c"
   ] {os = "openbsd" | os = "freebsd" | os = "macos"}
-  [make "world"]
-  [make "world.opt"]
+  [make "world.opt"] {os = "cygwin"}
+  [make "-j%{jobs}%" "world.opt"] {os != "cygwin"}
```

Comment:
Sorry for being late to this - you can use filters on the terms in a command as well, so the line above could be deleted and this changed to:
```suggestion
  [make "-j%{jobs}%" {os != "cygwin"} "world.opt"] 
```

I'm not particularly sure which more clearly conveys that parallelism is disabled on Cygwin!

Reply:
I think your proposal is clearer and more concise.